lib/cli.mjs

```diff
@@ -242,7 +243,7 @@ const flags = {
 		type: 'boolean',
 	},
 	fix: {
-		choices: [true, 'lax', 'strict'],
+		type: 'string',
```

Comment:
note

This avoids ``Error: Unknown value for flag `--fix`: `false`.``.
It also permits us to easily support `=0`/`=1` _if it were requested_ without having to maintain a list.
i.e. it used to be unofficially supported and could be considered a downgrade